Sources/Instrumentation/Sessions/SessionConstants.swift

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// Constants for OpenTelemetry session instrumentation.
+///
+/// Provides standardized attribute names and event types following OpenTelemetry
+/// semantic conventions for session tracking.
+///
+/// Reference: https://opentelemetry.io/docs/specs/semconv/general/session/
+public class SessionConstants {
+  // MARK: - OpenTelemetry Semantic Conventions
+  
+  /// Event name for session start events
+  public static let sessionStartEvent = "session.start"
+  /// Event name for session end events  
+  public static let sessionEndEvent = "session.end"
+  /// Attribute name for session identifier
+  public static let id = "session.id"
```

Comment:
There is existing conventions defined in https://github.com/open-telemetry/opentelemetry-swift/blob/main/Sources/OpenTelemetryApi/Trace/SemanticAttributes.swift which is auto-generated. It appears this may need to be updated. I'll follow up with how to do this.

Reply:
Hm, this looks like an issue we've neglected for some time; requiring the semantic attributes generation system to upgraded. disregard for now.